cmake_modules/BuildUtils.cmake

```diff
@@ -228,6 +228,7 @@ function(ADD_TERRIER_BENCHMARK REL_BENCHMARK_NAME)
     set(BENCHMARK_PATH "${EXECUTABLE_OUTPUT_PATH}/${BENCHMARK_NAME}")
     add_executable(${BENCHMARK_NAME} "${REL_BENCHMARK_NAME}.cpp")
     target_link_libraries(${BENCHMARK_NAME} ${TERRIER_BENCHMARK_LINK_LIBS})
+    set_target_properties(${BENCHMARK_NAME} PROPERTIES ENABLE_EXPORTS 1)
```

Comment:
Just curious what the purpose of this is.

Reply:
I think we can actually remove this. I probably have accidentally added this during one debugging, unless @17zhangw added this.

Reply:
I did not add this, so I will go ahead and remove it.